src/main/java/ch/jalu/configme/properties/EnumSetProperty.java

```diff
@@ -0,0 +1,31 @@
+package ch.jalu.configme.properties;
+
+import ch.jalu.configme.properties.types.EnumSetPropertyType;
+
+import java.util.Arrays;
+import java.util.EnumSet;
+import java.util.Set;
+
+/**
+ * Introduce an EnumSetProperty that goes along with EnumSetPropertyType.
+ *
+ * @param <E> enum
+ */
+public class EnumSetProperty<E extends Enum<E>> extends SetProperty<Set<E>> {
+
+    public EnumSetProperty(String path, Class<E> enumClass, EnumSet<E> defaultValue) {
+        super(path, new EnumSetPropertyType(enumClass), defaultValue);
+    }
+
+    public EnumSetProperty(String path, Class<E> enumClass, E... defaultValue) {
+        super(path, new EnumSetPropertyType(enumClass), castToEnumSet(enumClass, defaultValue));
+    }
+
+    private static <E extends Enum<E>> Set<E> castToEnumSet(Class<E> enumClass, E[] defaultValue) {
```

Comment:
I would call this `newEnumSet` to be in line with the `newSet` method of the parent class. "cast" implies the same object is being used